rel/files/riak-cs-debug

```diff
@@ -487,23 +555,44 @@ fi
 
 if [ 1 -eq $get_cfgs ]; then
     mkdir_or_die "${start_dir}"/"${debug_dir}"/config/.info
-    cd "${start_dir}"/"${debug_dir}"/config
 
-    # Use dump to execute the copy. This will provide a progress dot and
-    # capture any error messages.
-    dump cp_cfg cp -R "$cs_etc_dir"/*.conf* .
```

Comment:
Gathering riak-cs.conf and advanced.config is also helpful for CSEs to figure out a mistake on riak-cs.conf, isn't it?